tests/model_serving/model_server/conftest.py

```diff
@@ -154,3 +160,51 @@ def model_pvc(
 def skip_if_no_nfs_storage_class(admin_client: DynamicClient) -> None:
     if not StorageClass(client=admin_client, name=StorageClassName.NFS).exists:
         pytest.skip(f"StorageClass {StorageClassName.NFS} is missing from the cluster")
+
+
+@pytest.fixture(scope="class")
+def modelmesh_management_state(dsc_resource: DataScienceCluster) -> str:
+    return dsc_resource.instance.spec.components[DscComponents.MODELMESHSERVING].managementState
+
+
+@pytest.fixture(scope="class")
+def kserve_management_state(dsc_resource: DataScienceCluster) -> str:
+    return dsc_resource.instance.spec.components[DscComponents.KSERVE].managementState
```

Comment:
I'd prefer to have a single function/fixture that returns the management state for any component, I don't see the benefit in having to write the same one line function for each component

Reply:
removed the fixtures, calling directly where needed